docs/advanced_topics/formbuilder_routablepage_redirect.md

```diff
@@ -0,0 +1,50 @@
+# How to… use a redirect with Form builder to prevent double submission
+
+It is common for form submission HTTP responses to be a `302 Found` temporary redirection to a new page, so that a user who hits refresh doesn't submit their information again, because the URL has changed.
+
+This approach uses two contrib modules `wagtail.contrib.routable_page` and `wagtail.contrib.forms`.
+For a solution which redirects to an entirely different page, and doesn't require use of routablepages, see [Custom landing page redirect](../reference/contrib/forms/customisation.html#custom-landing-page-redirect).
```

Comment:
Not 100% sure but just trying to reword this to be a bit clearer that we are presenting what the tutorial does vs what is already documents.

```suggestion
Instead of showing the `render_landing_page` content on the POST response we can instead redirect to an entirely different URL with that same content managed by our `FormPage` class.

An alternative approach to this, that does not use `RoutablePageMixin` but instead redirects to an entirely different `Page` see [Custom landing page redirect](../reference/contrib/forms/customisation.html#custom-landing-page-redirect).
```

Reply:
Thanks, I've mostly adopted this change.